src/GlobalUtils.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2024 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+/**
+ * Global Utility Functions
+ */
+
+/**
+ * Generate the grid css.
+ * @param {number} columnStart - The number of the column that the grid item will start from.
+ * @param {number} columnSpan - The number of columns that the item will span across.
+ * @param {number} rowStart - The number of the row that the grid item will start from.
+ * @param {number} rowSpan - The number of rows that the item will span across.
+ * @return {String} - The grid css.
+ */
+function getGridStyle(columnStart: number, columnSpan: number, rowStart: number, rowSpan: number) {
```

Comment:
How about `generateGridStyle()` or `makeGridStyle()`?  This does not feel like a `get()` in the sense of retrieving a value.  Note that the documentation for the function is "Generate the grid css".

Reply:
Will change to `generateGridStyle()`.